Sources/Support/StoreMessagesHelper.swift

```diff
@@ -47,62 +59,47 @@ class StoreMessagesHelper {
         guard !self.showStoreMessagesAutomatically else {
             return
         }
-        // Compilation fails in older xcode versions with a similar issue to https://github.com/apple/swift/issues/57379
-        #if swift(>=5.8)
-        Task(priority: .medium) { [weak self] in
```

Comment:
This didn't need to create a `Task`.

Reply:
I suppose the value add may have been to set the priority explicitly, which can only be done with a separate Task?

Reply:
Oh I see. This method is already `async`. If we want to change the priority we should do it when we call this method in `PurchasesOrchestrator` /cc @tonidero